compiler/optimizer/OMRSimplifierTable.enum

```diff
@@ -682,6 +682,14 @@
 #define sbitpermuteSimplifierHandler dftSimplifier
 #define ibitpermuteSimplifierHandler dftSimplifier
 #define lbitpermuteSimplifierHandler dftSimplifier
+#define lcompressbitsSimplifierHandler dftSimplifier
+#define icompressbitsSimplifierHandler dftSimplifier
+#define scompressbitsSimplifierHandler dftSimplifier
+#define bcompressbitsSimplifierHandler dftSimplifier
+#define lexpandbitsSimplifierHandler dftSimplifier
+#define iexpandbitsSimplifierHandler dftSimplifier
+#define sexpandbitsSimplifierHandler dftSimplifier
+#define bexpandbitsSimplifierHandler dftSimplifier
```

Comment:
Does it never happen that the operands are constants (e.g., second operand being all zeroes or all ones) that will allow simplification?

Reply:
I have the same question, but I'm not clear what the answer was for this comment to be resolved.

Presumably, value propagation may leave us in a situation where both source and masks are constants which could be folded away.

Reply:
@0xdaryl see https://github.com/eclipse-omr/omr/pull/7259#issuecomment-2596735153

I've decided to leave implementing Tree Simplification and Value Propagation for a future change if it becomes clear that they are actually used in situations where these opts would be worthwhile.